flow360/component/simulation/web/draft.py

```diff
@@ -79,8 +79,9 @@ def submit(self) -> Draft:
         """
         Submit draft to cloud and under a given project
         """
-        draft_id = RestApi(DraftInterface.endpoint).post(self._request.model_dump(by_alias=True))
-        return Draft.from_cloud(draft_id["id"])
+        draft_meta = RestApi(DraftInterface.endpoint).post(self._request.model_dump(by_alias=True))
+        self._id = draft_meta["id"]
```

Comment:
again is `_id` used?

Reply:
same, see here: https://github.com/flexcompute/Flow360/blob/3d28af7ea2221967165e45a06aeb3be37e5e4cf9/flow360/component/resource_base.py#L176

Reply:
fixed in: https://github.com/flexcompute/Flow360/pull/526/commits/515cf1ea13850341d9b5ee25da3e4d494f3e7248